app/models/todo.rb

```diff
@@ -1,2 +1,5 @@
 class Todo < ActiveRecord::Base
+  def as_json
+    { id: id, title: title, is_completed: is_completed }
+  end
```

Comment:
I came across AMS while I was trying to set this up, but I wasn't sure if I was supposed to use them. I'll wait for Annie to talk about them first, though. Things make so much more sense when coming from her.